test/wasm_worker/wait32_notify.c

```diff
@@ -66,15 +65,14 @@ void* thread_main(void* arg) {
 
 
 EM_BOOL main_loop(double time, void *userData) {
-  if (addr == 1) {
+  if (addr == 3) {
     // Burn one second to make sure worker finishes its test.
```

Comment:
This PR seems like an improvement, but as a side note, this part of the test ("burn one second") looks fishy...

Reply:
I agree that is pretty nasty.. but its also pre-existing.  I looked into trying to fix that but it could be more involved so leaving it for now.